__tests__/OmnichannelChatSDKParallel.spec.ts

```diff
@@ -2374,6 +2374,158 @@ describe('Omnichannel Chat SDK, Parallel initialization', () => {
             expect(chatSDK.IC3Client).toBe(undefined);
         });
 
+        it('ChatSDK.endChat() should end conversation without calling backend close session', async () => {
```

Comment:
'ChatSDK.endChat() with optional params \'isSessionEnded\' set as \'true\' should not call sessionClose'